src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -445,13 +476,10 @@ private MinigameTask cloneChickenshock(final MinigameTask minigameTask, final St
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("chickenshock-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("chickenshock-backend not present");
-                    return new MinigameTask(Minigame.CHICKENSHOCK, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```

Comment:
In an exception, debug level? Also the backend name should be mentioned in the log.